WpfDesign.Designer/Project/Services/CreateComponentTool.cs

```diff
@@ -220,9 +239,9 @@ protected virtual DesignItem[] CreateItems(DesignContext context)
 		/// <summary>
 		/// Is called to create the item used by the CreateComponentTool.
 		/// </summary>
-		public static DesignItem CreateItem(DesignContext context, Type type)
+		public static DesignItem CreateItem(DesignContext context, Type type, object[] arguments = null)

```

Comment:
no, create an overload. even with a default value, it is a breaking change wich needs recompilation. 

the first method without arguments can then call the new one with